src/regex.js

```diff
@@ -10,12 +10,12 @@ export const makeAndTest = options =>
     testRegex
   )
 
-const anyWordToRegexp = _.flow(
+export const anyWordToRegexp = _.flow(
   _.words,
   _.join('|')
 )
 
-const wordsToRegexp = _.flow(
+export const wordsToRegexp = _.flow(
   _.words,
   _.map(x => `(?=.*${x})`),
```

Comment:
Positive lookahead is a fairly advanced RegEx concept. We could just surround the words with `.*` like this: 
```js
.*one.*two.*
```
which is equivalent in a does-it-match sense (not character consuming sense) to:
```js
(?=.*one)(?=.*two)
```

Reply:
@dubiousdavid but that would make it only match occurrences of the words that are in that order -- the existing code matches any order

Reply:
right - the any order thing matters

Reply:
Gotcha. Makes sense.